src/backend/aspen/workflows/nextstrain_run/build_config.py

```diff
@@ -47,6 +50,11 @@ def update_subsampling(self, config, subsampling):
             subsampling["country"]["max_sequences"] = 800
             subsampling["international"]["max_sequences"] = 200
 
+        # If there aren't any selected samples this is probably a scheduled run
+        # and we should use the reference sequences
+        if self.num_included_samples == 0:
+            del config["files"]["include"]
```

Comment:
This is a replacement for the `sed` command in the run_nextstrain_scheduled.sh script